content/docs/self-hosted-runners.md

```diff
@@ -341,7 +341,7 @@ are required to deploy EC2 instances.
 
 Click below to see credentials needed for supported compute providers.
 
-<toggle>
+<toggle height="250px">
 <tab title="AWS">
```

Comment:
LGTM, thanks @yathomasi 

Q: Does the CML team actually want to apply it here? It does look nicer IMO.

Reply:
Also, very minor 💅 but there's actually like a 1 px height difference between [?tab=AWS](https://cml-dev-toggle-tab-heig-fetgnj.herokuapp.com/doc/self-hosted-runners?tab=AWS) and [?tab=Azure](https://cml-dev-toggle-tab-heig-fetgnj.herokuapp.com/doc/self-hosted-runners?tab=Azure) 😅 I'm guessing that it's due to the emoji in the former's contents.

Reply:
:eyes: I'd prefer reverting this instance.

Reply:
> 👀 I'd prefer reverting this instance.

Sure, I added that for a reference.